mir-module/src/main/resources/xsl/response-facets.xsl

```diff
@@ -0,0 +1,234 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:encoder="xalan://java.net.URLEncoder"
+                xmlns:i18n="xalan://org.mycore.services.i18n.MCRTranslation"
+                xmlns:mcrxsl="xalan://org.mycore.common.xml.MCRXMLFunctions"
+                xmlns:xalan="http://xml.apache.org/xalan"
+                xmlns:fn="http://www.w3.org/2005/xpath-functions"
+                exclude-result-prefixes="i18n mcrxsl encoder xalan fn">
+
+  <xsl:param name="MIR.Response.Facet.Prefix.Classification"/>
+
+  <xsl:template name="facets">
+
+    <xsl:if test="/response/lst[@name='facet_counts']/lst[@name='facet_fields'] !=''">
```

Comment:
this test seems redundant with respect to the following xsl:for-each

Reply:
corrected